modules/auth/ldap/ldap.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"code.gitea.io/gitea/modules/log"
 
-	"gopkg.in/ldap.v3"
+	"github.com/go-ldap/ldap/v3"
```

Comment:
Is this deliberate?

Reply:
yes

Reply:
@zeripath https://github.com/go-ldap/ldap/blob/master/go.mod#L1